egs/aidatatang_200zh/ASR/local/compute_fbank_aidatatang_200zh.py

```diff
@@ -109,7 +109,12 @@ def get_args():
         default=80,
         help="""The number of mel bins for Fbank""",
     )
-
+    parser.add_argument(
+        "--speed-perturb",
+        type=bool,
```

Comment:
Please follow
https://github.com/k2-fsa/icefall/blob/00256a766921dd34a267012b0e2b8ff7d538f0e6/egs/librispeech/ASR/tdnn_lstm_ctc/asr_datamodule.py#L87
to use `str2bool`.

Reply:
thx! fixed this in the latest commit.